src/components/StepperHorizontal.js

```diff
@@ -0,0 +1,89 @@
+import { Box } from '@material-ui/core';
+import { Step } from '@material-ui/core';
+import { StepButton } from '@material-ui/core';
+import { Stepper } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const useStyles = makeStyles((theme) => ({
+  stepper: {
+    '& .MuiStepConnector-horizontal': {
+      left: 'calc(-50% + 10px)',
+      right: 'calc(50% + 10px)',
+      top: '10px',
+
+      '& .MuiStepConnector-lineHorizontal': {
+        borderTopWidth: '3px',
```

Comment:
I had some thoughts about it. Generally I don't think it is a good idea. It may be good idea but if design is prepared in such a way. It is good in my opinion to store some paddings, margin values but if they are consistent on design. So let's say designer uses 8px, 16px, 24px, 32px etc. You can then map it in variables and use them. But we have to agree on that with @triaslucia and which values we would like to use as standart ones. And then there is additional agreement if we want to stick with them strictly or we use them whetever possible but still get excceptions like here with 3px..